packages/twenty-front/src/modules/workflow/hooks/useWorkflowWithCurrentVersion.tsx

```diff
@@ -19,6 +19,7 @@ export const useWorkflowWithCurrentVersion = (
       id: true,
       name: true,
       statuses: true,
+      versions: true,
```

Comment:
I'm worried about getting all of the versions when I just want to know how many have ever been created for the workflow in order to conditionally display the "Discard Draft" button—we can discard a draft version only if it's not the first workflow version created.

Reply:
I don't know what the best way to solve this problem is according to Twenty's principle. However, a `versionsCount` field would be much better than my current solution. It would reduce the amount of transferred data and allocated JS objects in the browser. Depending on how we can implement this feature in the back-end, we could rely on a `SELECT COUNT(*)` operation instead of fetching all rows.

Reply:
That's good to know! Thank you, I'll make the change.

Reply:
Done